### PR TITLE
item table

### PR DESCRIPTION
**Pull request by hwjsnc**
Update to the item table schema (builds on the unpacker/packer changes made in PR #3).

### CODE REVIEW

File: schema/item_fr.py [hunk 1]
```diff
@@ -0,0 +1 @@
+item_en.py
```
Comment (nnguyen259) [inline schema/item_fr.py]: what would happen if we enable this in the unpacker?

Comment (hwjsnc) [reply schema/item_fr.py]: github doesn't show this well (or at all?), but the file is a symlink. On linux it works perfectly, but I have no idea what's the behaviour on windows.

Comment (nnguyen259) [reply schema/item_fr.py]: hmm I wonder what would happen if I create an .exe then run it

Comment (hwjsnc) [reply schema/item_fr.py]: Apparently newer versions of git can create symlinks on windows as well with a certain configuration option (core.symlinks = true). You could try that, or I could simply duplicate the schema file.

Comment (nnguyen259) [reply schema/item_fr.py]: yeah it just shows item_en.py when I pull it. So probably just duplicate it

Comment (hwjsnc) [reply schema/item_fr.py]: Done.

**Review by nnguyen259 — APPROVED** [review]

**Pull request merged**